app/views/users/show.html.haml

```diff
@@ -49,45 +49,46 @@
           .account-page-card-container
             .account-page-card-sub-container
               %h4.account-page-card-title API Key
-              %p.account-page-card-desc Your API Key can be used to access the NCBO API (REST) Services
+              %p.account-page-card-desc
+                = "Your API Key can be used to access #{portal_name} Web services"
               %p.apikey
                 = session[:user].apikey
             %a{:href => "javascript:navigator.clipboard.writeText('"+session[:user].apikey+"');"}
               = render partial: "shared/svgs/copy_icon"
         .account-page-api-documentation-link
-          %a{href: "https://data.bioportal.lirmm.fr/documentation", target: "_blank"}
+          %a{href: "https://data.#{portal_name}.lirmm.fr/documentation", target: "_blank"}
```

Comment:
need to be changed to use `$REST_API` variable, instead of `https://data.#{portal_name}.lirmm.fr`